core/tests/protractor_desktop/coreEditorAndPlayerFeatures.js

```diff
@@ -133,7 +133,7 @@ describe('Enable correctness feedback and set correctness', function() {
     await explorationEditorPage.navigateToMainTab();
     await explorationEditorMainTab.setStateName('First');
     await explorationEditorMainTab.setContent(await forms.toRichText(
-      'Select the right option.'));
+      'Select the right option.'), true);
```

Comment:
Are you changing the frontend code to have this element fade in? I don't see any mention in the PR description of UI changes. For reference, here's the `setContent` function:

https://github.com/oppia/oppia/blob/8a4ffd7ea179f21dd16916d26c6849566293f605/core/tests/protractor_utils/ExplorationEditorMainTab.js#L503

Reply:
I am frustrated now, this flake have been appeared from the develop, and like every time the flake become issue in my pr, I have not changed any code related to this.

Reply:
Hi @heyimShivam, please see https://github.com/oppia/oppia/pull/15806. This PR was merged 19 minutes ago. Does it solve the problem you are seeing with flakes? (We actually created it a couple of days ago, but it was delayed on reviewers.)

Reply:
hopefully, this solve the issue

Reply:
@seanlip we actually need some way to stop new flakes arriving in develop, otherwise Contributors will never get serious to solve their flakes.

Reply:
We do not have any change related to E2E files now in this PR, so don't think we need to make any changes.

Reply:
yes sean pr has solved that stateContentEditor flake